### PR TITLE
add team member Jörg

### PR DESCRIPTION
**Pull request by joergi-w**
Hopefully this links now automatically with the _Contact_ section on the LaRA page.

### CODE REVIEW

File: _people/joerg_winkler.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  github: joergi-w
+  linkedin: winklerj
+  keybase: joergi
+---
+
+As a PhD candidate at Freie Universität Berlin and member of the graduate school IMPRS-BAC at the Max Planck 
+Institute for Molecular Genetics I work on algorithms for the analysis of long non-coding RNA.
+I hold a Master of Science degree in Bioinformatics from University of Hamburg.
+
+I have developed the two applications *LaRA* for rapid structural RNA alignment and *MaRs* for RNA motif search.
```
Comment (joergi-w) [inline _people/joerg_winkler.md:17]: How can I insert a link here to the already existing LaRA page?

Comment (eseiler) [reply _people/joerg_winkler.md:17]: Can you rebase on the master? The LaRA page is not on this branch yet.
Regarding the linkage, we should find some easy way to also link people with apps, but I am no expert :)

**Comment by joergi-w** [comment]
Rebased on master.

**Comment by eseiler** [comment]
> Rebased on master.

Can you double check? It seems like you rebase onto your LaRA branch instead of master. It shows 4 commits and 2 of them add LaRA.

**Comment by joergi-w** [comment]
True, I accidentally rebased on origin/master and not upstream/master. 
I fixed it 👍🏽

File: _people/joerg_winkler.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+name: Jörg Winkler
+category: Team
+position: Research Assistant
```
Comment (marehr) [inline _people/joerg_winkler.md:4]:
```suggestion
position: PhD Student
```

Mitra wrote "PhD Student", is that fine with you?

Comment (joergi-w) [reply _people/joerg_winkler.md:4]: I prefer Research Assistant as this is the actual job title ("Wissenschaftlicher Mitarbeiter"), and I mention the PhD anyway in my first sentence. I think we are all more than just PhD students, no?

File: _people/joerg_winkler.md (hunk 1 above)
Comment (marehr) [reply _people/joerg_winkler.md]: Interesting Idea, we could automatically link-back / list all apps that an author has.

File: _people/joerg_winkler.md (hunk 2 above)
Comment (marehr) [reply _people/joerg_winkler.md:4]: Oki doki!

**Pull request merged**